### PR TITLE
Save coupon code against member

### PR DESCRIPTION
**Pull request by pezholio**
For https://github.com/theodi/shared/issues/625

### CODE REVIEW

File: db/migrate/20150520144034_add_coupon_to_member.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddCouponToMember < ActiveRecord::Migration
+  def change
+    add_column :members, :coupon, :string
```
Comment (pkqk) [inline db/migrate/20150520144034_add_coupon_to_member.rb]: It shouldn't affect that tests but why did this change have such a massive effect on the `db/schema.rb` file?

Comment (pezholio) [inline db/migrate/20150520144034_add_coupon_to_member.rb]: Good point. I'm using sqlite this end, so that might have something to do with it?

Comment (pezholio) [inline db/migrate/20150520144034_add_coupon_to_member.rb]: Looking at it, most of the changes are spacing. No idea why the `data_migrations` table has been dinged though. I thought that table was a Rails default anyway

Comment (pkqk) [inline db/migrate/20150520144034_add_coupon_to_member.rb]: Spacing is just rails terrible idea that lining things up is important. It was the column limits I was confused about but that might be sqlite.

Pretty sure the column limits don't equate to decimal digits directly.

**Pull request merged**